trade-finance-manager-ui/styles/fee-record-summary.scss

```diff
@@ -0,0 +1,16 @@
+.fee-record-summary {
+  @extend .block-header;
+  padding: govuk-spacing(4);
+  margin-bottom: govuk-spacing(8);
+
+  .govuk-body,
+  .govuk-link,
+  .govuk-back-link {
```

Comment:
Can only see `govuk-body` in `fee-record-summary`, in nunjucks, seems we only need govuk-body